core/arch/arm/include/ffa.h

```diff
@@ -278,13 +278,32 @@ struct ffa_mem_access_perm {
 	uint8_t flags;
 };
 
-/* Endpoint memory access descriptor */
-struct ffa_mem_access {
+/* Endpoint memory access descriptor up to version 1.1 */
+struct ffa_mem_access_1_1 {
 	struct ffa_mem_access_perm access_perm;
 	uint32_t region_offs;
 	uint64_t reserved;
 };
 
+/* Endpoint memory access descriptor from version 1.2 */
+struct ffa_mem_access_1_2 {
+	struct ffa_mem_access_perm access_perm;
+	uint32_t region_offs;
+	uint8_t impdef[16];
+	uint64_t reserved;
+};
+
+/*
+ * A common memory access descriptor where only the first two fields needs
+ * to be accessed.
+ */
+struct ffa_mem_access_common {
```

Comment:
`ffa_mem_access_x` to follow the same pattern as `ffa_mem_transaction`? (`ffa_mem_transaction_1_0`, `ffa_mem_transaction_1_1`, `ffa_mem_transaction_x`)